### ISSUE TITLE
Make "upload" and "create" more clear and explicit in UX

### ISSUE THREAD

**Issue by bnvk** [owncloud/core#21545]
Currently in ownCloud 8.2 upon logging into an instance, the only link or button for a user to perform the "upload" feature is tucked into a dropdown menu which is accessed by the `+` icon. Upon clicking this a user is presented with:
- Upload
- Text File
- Folder

I managed to figure out "how to upload a file" but it took about 1 minute longer than I think it should to find such a common user interaction. Additionally, the terse / short wording of the items in the menu are a bit ambiguous, IMHO, considering what they fundamentally do are very different
- Upload (selects a file on a users computer)
- Text File & Folder (both create a new item on the server)

For the sake of being overly explicit and clear (for the less savvy & inclined users), and more semantic in terms of interactions, I propose the following buttons:

[ Upload From Your Computer ]   [ Create On ownCloud ] 

The "upload" button could simply trigger the file select dialogue, while the "create" could show a dropdown, like the current. There seems to be plenty of UI space for this more explicit buttons, and I'd wager they'd perform better in users tests!

@jancborchardt what say you about this?

**Comment by MorrisJobke** [comment]
cc @owncloud/designers

**Comment by jancborchardt** [comment]
We used to have it like that in previous ownCloud versions. That is, a separate button for »New« and one right next to it with an upload icon:
![owncloud](https://cloud.githubusercontent.com/assets/925062/12588843/e7bda018-c45a-11e5-9d89-39a7c47844c9.png)
(Very similar to Google Drive for example.)

However we changed this due to several reasons:
- Separation between the two doesn’t really matter. It’s creating something new. That way there is one initial way to add. (Except drag&drop of course.)
- Space on mobile is limited. With the new single menu it works nicely.
- Taking cues from other systems like iOS and Android where there is one simple [+] button for adding stuff, be it from your phone or creating a new folder.
